File: java/src/main/java/id/my/niznet/tugas2/Segitiga.java
package id.my.niznet.tugas2;

public class Segitiga implements BangunDatar {
    private double alas, tinggi;

    public Segitiga(double alas, double tinggi) {
        this.alas = alas;
        this.tinggi = tinggi;
    }

    @Override
    public double luas() {
        return (alas * tinggi) / 2;
    }

    @Override
    public double keliling() {
        return 0;
    }
}

File: kotlin/src/main/kotlin/id/my/niznet/tugas2/Segitiga.kt
package id.my.niznet.tugas2

class Segitiga(private val alas: Double, private val tinggi: Double) : BangunDatar {
    override fun luas(): Double {
        return alas * tinggi / 2
    }

    override fun keliling(): Double {
        return 0.0
    }
}